src/web_interface/templates/show_analysis.html

```diff
@@ -112,7 +112,11 @@
         {# Header section #}
         <div class="header mb-4" style="word-wrap: break-word">
             <h3>
-                {{ firmware.uid | replace_uid_with_hid(root_uid=root_uid) | safe }}<br />
+                {{ firmware.uid | replace_uid_with_hid(root_uid=root_uid) | safe }}
+                {% if link_target %}
+                    ({{ link_target | safe }})
```

Comment:
This is not `safe`:

```sh
printf "\n" > '<script>alert("injection");'
ln -s \<script\>alert\(\"injection\"\)\; foobar

tar -cvf x.tar ./*
```
While this does not run `altert`, it still breaks the whole site.